packages/core-connector-interface/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "strict": true,
```

Comment:
Why is this in the logging PR?

Reply:
Went through all packages to fetch their latest `package.json` / `tsconfig.json` etc and unified them.

Reply:
It's not logging related?

Reply:
Yes!

Reply:
Yes!